Quoridor.Model/Graph.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+
+namespace Quoridor.Model {
+    class Graph {
+        private int size;
+        private int[,] edges;
+        private LinkedList<int>[] adjacencyList;
+
+        public Graph(int _size) {
+            this.size = _size * _size;
+            this.edges = new int[2 * _size * (_size - 1), 2];
+
+            int idW = 0;
```

Comment:
If "idW" is "id of the wall" then, maybe, it will be better to name it "wallId"?